scripts/run_mypy_checks.py

```diff
@@ -769,37 +774,68 @@ def install_third_party_libraries(skip_install):
         install_third_party_libs.main()
 
 
-def get_mypy_cmd(files):
+def get_mypy_cmd(files, install_globally):
```

Comment:
```suggestion
def get_mypy_cmd(files, using_global_mypy):
```

This function doesn't install anything so `install_*` variable is bit off.

Reply:
Done!